cpp/CMakeLists.txt

```diff
@@ -150,22 +153,22 @@ if (${USE_STD_MAP} STREQUAL "OFF")
 endif ()
 
 # Add protoc (Protocol Buffers compiler) target.
-set (RESOURCES_DIR "${CMAKE_SOURCE_DIR}/../resources")
+set (RESOURCES_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../resources")
```

Comment:
Is this variable name change specific to your setup?

Reply:
CMAKE_CURRENT_SOURCE_DIR is cmake specific and must be used in this context as it correctly specifies the current folder of the existing cmake file. CMAKE_SOURCE_DIR represents the current folder of the top level cmake file.